recipes/libtool/all/conanfile.py

```diff
@@ -220,22 +219,21 @@ def package_info(self):
             if self.settings.os == "Linux":
                 self.cpp_info.system_libs = ["dl"]
 
+        # Define environment variables such that libtool m4 files are seen by Automake
+        libtool_aclocal_dir = os.path.join(self._datarootdir, "aclocal")
+        self.output.info("Appending ACLOCAL_PATH env: {}".format(libtool_aclocal_dir))
+        self.output.info("Appending AUTOMAKE_CONAN_INCLUDES environment variable: {}".format(libtool_aclocal_dir))
```

Comment:
```suggestion
        self.output.info("Appending ACLOCAL_PATH env: {libtool_aclocal_dir}")
        self.output.info("Appending AUTOMAKE_CONAN_INCLUDES environment variable: {libtool_aclocal_dir}")
```
f-strings improve legibility (IMHO)

Reply:
I think these messages can be removed because in conan v2 they are not always appended to environment depending on context.